website/docs/r/waf_security_rule_exception.html.markdown

```diff
@@ -0,0 +1,125 @@
+---
+layout: "incapsula"
+page_title: "Incapsula: waf-security-rule-exception"
+sidebar_current: "docs-incapsula-resource-waf-security-rule-exception"
+description: |-
+  Provides a Incapsula WAF Security Rule Exception resource.
+---
+
+# incapsula_site
+
+Provides a Incapsula WAF Security Rule Exception resource.  Important to note that based on the rule_id the exception is being created for, that there are rule specific parameters that apply to each.  The example resources listed below include all of the supported resources for each rule_id or rule type, although it is not required to use all listed parameters when creating an exception. Exception parameters are optional but at least one is required.
```

Comment:
Single space after the period. Do a newline after the first sentence to keep parity with the other docs.